web/src/repositories/ProjectRepository.js

```diff
@@ -3,6 +3,13 @@ import Repository from '@/repositories/Repository'
 const resource = 'doc'
 export default {
 
+  /**
+  * Get Config
+  */
+  getConfig() {
+    return Repository.get(`${Repository.defaults.baseURL}/config.json`)
```

Comment:
I would suggest to await the result here and use a try catch around it beause if the config file is
not there this will result in a 404 error :)

Reply:
it will be probably always available, but better safe than sorry :+1: